torch/onnx/symbolic_opset9.py

```diff
@@ -3801,6 +3801,11 @@ def _any(g, *args):
     input_sum = symbolic_helper._reducesum_helper(
         g, input, axes_i=dim, keepdims_i=keepdim
     )
+
+    if dim is None and keepdim == 0:
+        return g.op(
+            "Squeeze", gt(g, input_sum, g.op("Constant", value_t=torch.LongTensor([0])))
+        )
     return gt(g, input_sum, g.op("Constant", value_t=torch.LongTensor([0])))
```

Comment:
I think the real issue is that in Greater the value_t actually broadcast the output, so instead of adding Squeeze, the fix could be changing 
```python
return gt(g, input_sum, g.op("Constant", value_t=torch.tensor(0)))
```

Reply:
Indeed, I overlooked how the constant, which has `dim=1`, is created previously.